app/Http/Controllers/ReportsController.php

```
@@ -419,23 +354,181 @@ function splitAmountInTwo($amount) {
 
 
-  
 
 
+
+    private function createDynamicPayments($sub)
+    {
+        $rem = intVal($sub['rem']);
+        $payments = [];
+        $balance = $sub['amount_to_depreciate'];
+        $newBal = 0;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$newBal'. (Unused Code Rules)

(UnusedLocalVariable)